src/Microsoft.Identity.Web.OWIN/AppBuilderExtension.cs

```diff
@@ -64,13 +64,13 @@ public static class AppBuilderExtension
             string clientId = configuration.GetValue<string>($"{configurationSection}:ClientId");
             string audience = configuration.GetValue<string>($"{configurationSection}:Audience");
             string authority = instance + tenantId + "/v2.0";
-            TokenValidationParameters tokenValidationParameters = new TokenValidationParameters
+            TokenValidationParameters tokenValidationParameters = new()
             {
                 ValidAudience = audience ?? clientId,
                 IssuerValidator = AadIssuerValidator.GetAadIssuerValidator(authority).Validate,
```

Comment:
~Interesting. It seems we have it here already?~
Actually this was already done in `AddMicrosoftIdentityWebApi` (as expected)

What you have done is in AddMicrosoftIdentityWebApp (for the validation of the ID token), and it's right